analysis/script/travis-build.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
 cd analysis
-./gradlew build integrationTest
+./gradlew build
```

Comment:
Could this step have influence on our sonarcloud results?

Reply:
No. This script runs when we enter the deploy stage. The deploy stage is independent from the sonar stage.